jekyll/_cci2/hello-world.adoc

```diff
@@ -0,0 +1,531 @@
+
+[source,yaml]
+----
+version: 2.1
+
+orbs:
+  win: circleci/windows@5.0.0 # The Windows orb gives you everything you need to start using the Windows executor.
+
+jobs:
+  hello-job:
+    executor:
+      name: win/default # executor type
+      size: "medium" # resource class, can be "medium", "large", "xlarge", "2xlarge", defaults to "medium" if not specified
+
+    steps:
+      # Commands are run in a Windows virtual machine environment
+      - checkout
+      - run: Write-Host 'Hello, Windows'
+
+workflows:
+  my-workflow:
+    jobs:
+      - hello-job
+----
+--
++
+[.tab.hellocloud.GPU]
+--
+NOTE: The GPU execution environment is available on the link:https://circleci.com/pricing/[Scale] plan.
+
+The job `hello-job` spins up a GPU-enabled virtual machine using the machine executor. GPU images are available for link:/docs/configuration-reference#available-windows-gpu-image[Windows] and link:/docs/configuration-reference#available-linux-gpu-images[Linux]. Refer to link:/docs/using-gpu[Using the GPU execution environment] page for more information.
+
+[source,yaml]
+----
+version: 2.1
+
+jobs:
+  hello-job:
+    machine:
+      image: ubuntu-2004-cuda-11.4:202110-01
+    steps:
+      - checkout # check out the code in the project directory
+      - run: echo "hello world" # run the `echo` command
+
+workflows:
+  my-workflow:
+    jobs:
+      - hello-job
+----
+--
++
+[.tab.hellocloud.Arm]
+--
+The job `hello-job` spins up an [Arm (Linux) virtual machine] using the machine executor. Refer to link:/docs/using-arm[Using the Arm execution environment] page for more information.
+
+[source,yaml]
+----
+version: 2.1
+
+jobs:
+  hello-job:
+    machine:
+      image: ubuntu-2004:202101-01
+    resource_class: arm.medium
+    steps:
+      - checkout # check out the code in the project directory
+      - run: echo "hello world" # run the `echo` command
+
+workflows:
+  my-workflow:
+    jobs:
+      - hello-job
+----
+--
+
+. Commit and push the changes to your VCS.
+
+. Go to the **Projects** page in the CircleCI web app, then click the **Set Up Project** button next to your new project. If you do not see your project, make sure you have selected the correct organization at the top-left of the web app. See the link:/docs/introduction-to-the-circleci-web-app#organizaiton-switching[Organization Switching] for more information.
+
+. Follow the steps in the pop-up to tell CircleCI to use the `config.yml` file you just created to trigger your first pipeline. Clicking through to the `hello-job` and then selecting the `echo "hello world" step` will show you `hello world` in the console.
++
+image:hello-world-app.png[Screenshot showing hello world in the job step output]
+
+**Tip:** If you get a `No Config Found` error, it may be that you used `.yaml` file extension. Be sure to use `.yml` file extension to resolve this error.
+
+[#echo-hello-world-server-4]
+== Echo hello world (server v4.x)
```

Comment:
Do we also need the server version in the section heading? (I am assuming it's because we can't have duplicate heading values within the same document, even if the anchors/ids are unique ..?)

Reply:
I had left it in because I am not yet sure what the default state would be when tagging is implemented (e.g. default all visible until you select an option, or default just Cloud is visible), and these changes will go in pre-tagging so people need to be able to see what each section is for.

Reply:
Gotcha! Not sure how I feel about parentheses in the heading, do you think it would be better to remove them and say "on"/"in" instead..?

Reply:
Yeah good idea, will do!